crates/swc/tests/error_msg.rs

```diff
@@ -36,8 +36,8 @@ fn swcrc_simple() {
     println!("{}", f);
 }
 
-#[testing::fixture("tests/errors/**/input.js")]
-#[testing::fixture("tests/errors/**/input.ts")]
+#[testing::fixture("tests/errors/**/no-prototype-builtins/**/input.js")]
```

Comment:
Please revert this

Reply:
If you want to run specific test only, you can provide argument to `cargo test` like 

`cargo test foo` to run only tests that contains `foo` in the test name

Reply:
> Please revert this

Oh, sorry. I'm forgot to discard unnecessary changes